src/webrtc/callFeed.ts

```diff
@@ -55,6 +56,7 @@ type EventHandlerMap = {
     [CallFeedEvent.LocalVolumeChanged]: (localVolume: number) => void;
     [CallFeedEvent.VolumeChanged]: (volume: number) => void;
     [CallFeedEvent.Speaking]: (speaking: boolean) => void;
+    [CallFeedEvent.Disposed]: (disposed: boolean) => void;
```

Comment:
I don't entirely understand why the `disposed: boolean` argument exists

Reply:
I was thinking of an SFU use case but I'd actually do the thing differently in the end anyway :sweat_smile: